From 95607140fba8ac876346ca6924c4549de8cecf64 Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Mon, 14 Nov 2022 19:04:43 -0800
Subject: [PATCH 187/196] Bump version to 4.3.3 (#292)

* Supports "Numbers" upgrade
* Updates to Bitcoin ABC 0.24.11
* Updates block rewardee addresses
---
 CMakeLists.txt                | 2 +-
 contrib/aur/lotus-qt/PKGBUILD | 2 +-
 contrib/aur/lotus/PKGBUILD    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index adcf60292..4514f697f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 )
 
 project(lotus
-	VERSION 3.3.3
+	VERSION 4.3.3
 	DESCRIPTION "Lotus is a full node implementation of the Lotus protocol."
 	HOMEPAGE_URL "https://www.givelotus.org"
 )
diff --git a/contrib/aur/lotus-qt/PKGBUILD b/contrib/aur/lotus-qt/PKGBUILD
index 6ec533146..c29d6b034 100644
--- a/contrib/aur/lotus-qt/PKGBUILD
+++ b/contrib/aur/lotus-qt/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotus-qt
-pkgver=1.0.2
+pkgver=4.3.3
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-cli, lotus-tx, lotus-seeder and lotus-qt"
 arch=('i686' 'x86_64')
diff --git a/contrib/aur/lotus/PKGBUILD b/contrib/aur/lotus/PKGBUILD
index 476d9a2c5..eb19c9d4c 100644
--- a/contrib/aur/lotus/PKGBUILD
+++ b/contrib/aur/lotus/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotusd
-pkgver=3.3.3
+pkgver=4.3.3
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-tx, lotus-seeder and lotus-cli"
 arch=('i686' 'x86_64')
-- 
2.39.5 (Apple Git-154)

